external/runtime/runtime.depproj

```diff
@@ -11,13 +11,20 @@
     <PackageReference Include="Microsoft.DiaSymReader.Native" Version="1.7.0" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <!-- These needs to be replaced once we have an official build -->
```

Comment:
nit: 
```suggestion
    <!-- These needs to be replaced once we have an stable build of 3.0 -->
```

Technically we might be able to remove them once we have stable build of 3.0 and just use the implicit framework reference from the SDK.  We'd need to see if we can make that work.